contentcuration/contentcuration/frontend/shared/views/files/Thumbnail.vue

```diff
@@ -97,6 +106,10 @@
         type: [Number, String],
         default: 'none',
       },
+      printing: {
```

Comment:
I made a mixin for this that I am using in ExpandableList, DetailsRow, and Details - instead of using a prop, I am using the provide/inject syntax, so that we can just set `printing: true` at the parent printing component, and have it picked up as context in any child component - saves us having to pass it down as props.